CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [0.12.1-alpha.0](https://github.com/chanzuckerberg/edu-design-system/compare/v0.12.0...v0.12.1-alpha.0) (2022-04-27)
+
+* chore(text): reintroduce spacing variant for text
+
```

Comment:
Do we want the alpha versions in here? I don't think we've been adding them in the past.

Reply:
I think we could be keeping these, and historically are on npm. What are your thoughts?

Reply:
Personally I don't want to feel like I have to add every alpha version when we're just using them for testing. Seems like a hassle, and I don't see the benefit of having it logged here.

Reply:
Ahh makes sense, will remove